sync/src/main/java/org/bf2/sync/ManagedKafkaSync.java

```diff
@@ -167,7 +176,12 @@ public boolean changed(ManagedKafka remote, ManagedKafka existing) {
             return true;
         }
         ManagedKafka remoteCopy = secretManager.removeSecretsFromManagedKafka(remote);
-        return !remoteCopy.getSpec().equals(existing.getSpec()) || !Objects.equals(existing.getMetadata(), remoteCopy.getMetadata());
+        // will always be non-null due to the id/placement requirements
+        Map<String, String> annotations = new HashMap<>(existing.getMetadata().getAnnotations());
+        annotations.keySet().removeAll(DATA_PLANE_ANNOTATIONS);
```

Comment:
Are we removing all the data-plane annotations here to detect if any control plane annotations have changed?

Reply:
Yes.  They aren't doing that yet we're just future-proofing.  The placement id annotation can change, but the exiting managedkafka should be deleted before they make that change.